### PR TITLE
Adds live migration know issue to release notes

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.16

Issue:
https://issues.redhat.com//browse/OCPBUGS-35316

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3137,6 +3137,12 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
 
 * If you delete a `SriovNetworkNodePolicy` resource for a virtual function with a `vfio-pci` driver type, the SR-IOV Network Operator is unable to reconcile the policy. As a consequence the `sriov-device-plugin` pod enters a continuous restart loop. As a workaround, delete all remaining policies affecting the physical function, then re-create them. (link:https://issues.redhat.com/browse/OCPBUGS-34934[*OCPBUGS-34934*])
 
+* The limited live migration method stops when the `ConfigMap` MTU object is missing and returns the following error: `NetworkTypeMigrationFailed. Failed to process SDN live migration (configmaps "mtu" not found)`.
++
+In most cases, the `ConfigMap` MTU object should be created by the `mtu-prober` job during installation. However, if the cluster was upgraded from an early release, for example, {product-title} 4.4.4, the `ConfigMap` MTU object might be absent.
```
Comment (maxwelldb) [inline release_notes/ocp-4-16-release-notes.adoc:3142]: Could make the first sentence more active.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -3137,6 +3137,12 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
...
++
+As a temporary workaround, you can manually create the `ConfigMap` MTU object before starting the limited live migration process. This issue will be fixed in a future version of {proudct-title}. (link:https://issues.redhat.com//browse/OCPBUGS-35316[*OCPBUGS-35316*])
```
Comment (maxwelldb) [inline release_notes/ocp-4-16-release-notes.adoc:3144]:
```suggestion
As a temporary workaround, you can manually create the `ConfigMap` MTU object before starting the limited live migration process. This issue will be fixed in a future version of {product-title}. (link:https://issues.redhat.com//browse/OCPBUGS-35316[*OCPBUGS-35316*])
```

Comment (maxwelldb) [reply release_notes/ocp-4-16-release-notes.adoc:3144]: "This issue will be fixed in a future version..."

Unless there's a day one patch that's ready to go, you can omit this. It's an unactionable prediction. 🔮

**Review by maxwelldb — COMMENTED** [review]
Looks pretty good! Be sure to fix the attribute name typo.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -3143,6 +3143,12 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
 
 * There is no log pruning available for azcopy (underlying tool running copy jobs) in Azure, so this might eventually lead to filling up a root device of the controller pod, and you have to manually clean this up. (link:https://issues.redhat.com/browse/OCPBUGS-35980[*OCPBUGS-35980*])
 
+* The limited live migration method stops when the `ConfigMap` MTU object is missing and returns the following error: `NetworkTypeMigrationFailed. Failed to process SDN live migration (configmaps "mtu" not found)`.
```
Comment (pliurh) [inline release_notes/ocp-4-16-release-notes.adoc:3146]: 1. CNO will not report any error in this case. The live migration will just get stuck at the last step, and can't be finished.
2. The name of the configmap is 'mtu', not 'MTU'. We shall also mention that the configmap is in the namespace 'openshift-network-operator'.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 4]
```diff
@@ -3143,6 +3143,12 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
...
++
+In most cases, the `ConfigMap` MTU object is created by the `mtu-prober` job during installation. However, if the cluster was upgraded from an early release, for example, {product-title} 4.4.4, the `ConfigMap` MTU object might be absent.
++
+As a temporary workaround, you can manually create the `ConfigMap` MTU object before starting the limited live migration process. (link:https://issues.redhat.com//browse/OCPBUGS-35316[*OCPBUGS-35316*])
```
Comment (pliurh) [inline release_notes/ocp-4-16-release-notes.adoc:3150]: Here's an example of the configmap. 
Note: the MTU value shall be aligned with the MTU of the node interface.
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: mtu
  namespace: openshift-network-operator
data:
  mtu: "1500"
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 5]
```diff
@@ -3143,6 +3143,26 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
 
 * There is no log pruning available for azcopy (underlying tool running copy jobs) in Azure, so this might eventually lead to filling up a root device of the controller pod, and you have to manually clean this up. (link:https://issues.redhat.com/browse/OCPBUGS-35980[*OCPBUGS-35980*])
 
+* The limited live migration method stops when the `mtu` parameter of a `ConfigMap` object in the `openshift-network-operator` namespace is missing.
++
+In most cases, the `mtu` field of the `ConfigMap` object is created by the `mtu-prober` job during installation. However, if the cluster was upgraded from an early release, for example, {product-title} 4.4.4, the `ConfigMap` object might be absent.
++
+As a temporary workaround, you can manually create the `ConfigMap` object before starting the limited live migration process. For example:
```
Comment (weliang1) [inline release_notes/ocp-4-16-release-notes.adoc:3150]: @stevsmit https://issues.redhat.com//browse/OCPBUGS-35316 is verified when the `mtu` parameter of a `ConfigMap` object in the `openshift-network-operator` namespace is missing. This is not an issue any more.

**Pull request merged**